src/components/Spinner.tsx

```diff
@@ -28,16 +20,12 @@ export function Spinner({
     >
       <div
         className={cn(
-          "border-6 mx-2 animate-spin rounded-full border-2 border-solid border-l-slate10",
+          "border-6 mx-2 animate-spin rounded-full border-2 border-solid border-l-primary border-brown12",
```

Comment:
should use the colors from the theme. FWIW I think this exact component is used in cowamm and I fixed this there

Reply:
https://github.com/bleu-fi/monorepo/pull/640